internal/controllers/suite_test.go

```diff
@@ -74,22 +123,20 @@ func TestMain(m *testing.M) {
 	}
 
 	var err error
-	cfg, err = testEnv.Start()
+	config, err = testEnv.Start()
 	utilruntime.Must(err)
-	if cfg == nil {
+	if config == nil {
 		log.Panic("expected cfg to not be nil")
 	}
 
 	rm := meta.NewDefaultRESTMapper(nil)
-	cfgGetter, err := helmclient.NewActionConfigGetter(cfg, rm)
+	cfgGetter, err := helmclient.NewActionConfigGetter(config, rm)
 	utilruntime.Must(err)
-	acg, err = helmclient.NewActionClientGetter(cfgGetter)
+	helmClientGetter, err = helmclient.NewActionClientGetter(cfgGetter)
 	utilruntime.Must(err)
 
-	mgr, err := manager.New(cfg, manager.Options{})
```

Comment:
Why remove manager? This seems weird. 

Also most of the diff here is in the naming - for ex (cfg to config, act to helmClientGetter). Its fine, but in order to keep the diffs low, can we revert them as before?

Reply:
Do we actually use the manager anywhere? I wouldn't expect to need a manager in a unit test.